lib/dry/cli/command_context.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require 'ostruct'
+
+module Dry
+  class CLI
+    # CommandContext
+    #
+    # @since 0.6.x
+    class CommandContext < OpenStruct
```

Comment:
Please *do not* use `OpenStruct` for this. It responds to any method and returns `nil` even when there's no corresponding attribute. There are very few cases where `OpenStruct` is a good choice and this isn't one. Even if we want to support arbitrary attributes it's still better to have an internal hash + method_missing based on `key?` checks.

Reply:
This implementation of Context is based on https://github.com/hanami/hanami/blob/master/lib/hanami/cli/commands.rb#L63 internal hanami Context. I believe I can improve it a little bit. sure, will fix it (was thinking about it too)

Reply:
I currently replaced with primitive implementation, just for now. Later, when I'll start adding generation abilities, it will evolve